cmd/tetragon/main.go

```diff
@@ -133,6 +135,13 @@ func tetragonExecute() error {
 		log.Fatal(err)
 	}
 
+	option.Config.TracingPolicyDir = filepath.Join("/", option.Config.TracingPolicyDir)
```

Comment:
The Abs if the path is not absolute it will be joined with current directory, and I opted for defensive. If a user by accident points to a relative path we explicitly join with / then force the Abs below, making it fail later during walkdir since those tracing policies are bpf programs that could impact the system... So to ensure that users are specifying their correct paths (including us developers ;-) )

Reply:
Let's check if user provided an absolute path, (IsAbs) and then reject the file afterwards.

Reply:
done, thanks ;-)